Resources/Prototypes/SS220/Entities/Clothing/Neck/iron_squad_cloaks.yml

```
@@ -3,7 +3,7 @@
   id: ClothingCloakIronSquadCMO
   name: плащ Главного Врача Железного отряда
   description: Этот плащ не только нарушает любые санитарные нормы, но и наделяет таким авторитетом надевшего его, что о первом никто не смеет упомянуть.
```

```suggestion
  description: Этот плащ не только нарушает любые санитарные нормы, но и наделяет владельца таким авторитетом, что его никто не посмеет обсуждать за спиной.
```
о ком первом?
упомянуть о чём? как?